scripts/x-charts.exports.json

```diff
@@ -107,7 +107,7 @@
   { "name": "cheerfulFiestaPaletteDark", "kind": "Variable" },
   { "name": "cheerfulFiestaPaletteLight", "kind": "Variable" },
   { "name": "ComputedPieRadius", "kind": "Interface" },
-  { "name": "ContinuouseScaleName", "kind": "TypeAlias" },
+  { "name": "ContinuousScaleName", "kind": "TypeAlias" },
```

Comment:
I could also do something like `export ContinuousScaleName as ContinuouseScaleName` to avoid the breaking change, but it looks more problematic than the breaking change it tries to fix

Reply:
Oh yeah, to it also looks like a small BC that's not worth creating extra redundant technical debt for. 🙈